src/index.js

```diff
@@ -0,0 +1,92 @@
+import './style.css';
+// grabing up the neede HTML elements to be used in JS codes.....😎😊😉😎............
+
+const btn = document.querySelector('form');
+const listInput = document.querySelector('#lists-input');
+const cleanAllDone = document.querySelector('#clearDones');
+const listHolder = document.querySelector('#list-holder');
+
+// *********Empty array for storing datas ***********
+let listArray = [];
+
+// Store the listArray in Local Storage when the form is submitted....😎😊😉😎............
+
+class UI {
+  static displayData() {
+    const datas = listArray.map((item) => `<div class="col-12" id="list">
+                <p class="checkboxP"> 
+                <input class="form-check-input" type="checkbox" value="" id="defaultCheck">
+                <span id="description">${item.discription}</span>
+                </p>
+                <span><i class="fa fa-ellipsis-v" aria-hidden="true"></i></span>
+            </div>`);
+    listHolder.innerHTML = (datas).join(' ');
+  }
+  // cleaning the input field after submition process....😎😊😉😎............
+
+  static cleanInputs() {
+    listInput.value = '';
+  }
+}
```

Comment:
- [ ] Also I would suggest you please create the new JavaScript file for the new functionality. It helps you to find the bug easily and help to read the code easily.

Reply:
ok Thank you sir